tests/cli/upload/test_cli_upload.py

```diff
@@ -17,10 +17,16 @@ def test_all_samples_data_analysis(analysis_store):
     assert LinkHelper.all_samples_data_analysis(family_obj.links, ['mip'])
 
 
-def test_all_samples_are_wgs(analysis_store):
+def test_all_samples_check_analysis(analysis_store):
```

Comment:
I suggest using [GivenWhenThen](https://martinfowler.com/bliki/GivenWhenThen.html) when writing tests improve the maintainability of the tests, i.e. we are writing in clear text what we are aiming to do/test